.golangci.yml

```diff
@@ -176,7 +176,7 @@ linters-settings:
   lll:
     # max line length, lines longer will be reported. Default is 120.
     # '\t' is counted as 1 character by default, and can be changed with the tab-width option
-    line-length: 120
```

Comment:
why change this rule

Reply:
Because `ChargingMethod string `json:"chargingMethod,omitempty" yaml:"chargingMethod" bson:"chargingMethod" mapstructure:"chargingMethod"`` is too long.

Reply:
rollback to `120` and add `// no-lint` after that line.